python/paddle/v2/reader/tests/decorator_test.py

```diff
@@ -144,6 +144,36 @@ def mapper(x):
                         for idx, e in enumerate(result):
                             self.assertEqual(e, mapper(idx))
 
+class TestPipeReader(unittest.TestCase):
+    def test_pipe_reader(self):
+        def simple_parser(lines):
+            return lines
+        import tempfile
+
+        records = [str(i) for i in xrange(5)]
+        temp = tempfile.NamedTemporaryFile()
+        try:
+            with open(temp.name, 'w') as f:
+                for r in records:
+                    f.write('%s\n' % r)
+
+            cmd = "cat %s" % temp.name
+            #reader = paddle.v2.reader.pipe_reader(cmd,
+            reader = pipe_reader(cmd,
+                                                  simple_parser,
```

Comment:
Indentation error?

Reply:
Delete the unused comment and it's just for test.